packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -0,0 +1,210 @@
+.rt-DataListRoot {
+  display: grid;
+  grid-template-columns: auto 1fr;
+}
+
+.rt-DataListItem {
+  position: relative;
+  box-sizing: border-box;
+  grid-column: span 2;
+}
+
+.rt-DataListLabel {
+  color: var(--gray-a11);
+  box-sizing: border-box;
+  display: block;
+  white-space: nowrap;
+}
+
+.rt-DataListData {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
Let's put this margin reset into a related rule in `reset.css` and use the `rt-reset` class instead.
(e.g. check `link.tsx` as an example, `rt-reset` is applied separately to keep the component styles focused just on adding new styles)